web-app/src/index.js

```diff
@@ -35,10 +36,13 @@ const configureStore = () => {
 
 const isAuthenticated = () => {
   if (!authenticated()) {
-    return <Redirect to="/app" />;
+    if (window.location.pathname !== '/landingpage' && window.location.pathname !== '/develop' && window.location.pathname !== '/app' && window.location.pathname !== '/login' && window.location.pathname.substring(0, 25) !== '/login/callback/facebook/' && window.location.pathname !== '/reset-password' && window.location.pathname.substring(0, 16) !== '/reset-password/' && window.location.pathname !== '/sign-up' && window.location.pathname !== '/you-made-it' && window.location.pathname !== '/start') {
```

Comment:
@RobinMalfait Thanks for the improvement. 

@DemianD I agree; in that case there is need for a ReactJS component such ass the <App /> for the onepager and landingpage. Didn't know GitHub Pages doesn't support Laravel; so that's the only other possibility I assume - using ReactJS. But to be online asap this could be used and you can make it ReactJS afterwards if you have time.